File: app/src/main/java/com/avanade/testesantander2/loginScreen/LoginModel.java
package com.avanade.testesantander2.loginScreen;

import com.avanade.testesantander2.ApiPostLoginResponse;
import com.avanade.testesantander2.Erro;
import com.avanade.testesantander2.UserAccount;
import com.google.gson.annotations.Expose;

public class LoginModel {

}

class LoginViewModel{
    // Response da API
    // LoginResponse[UserAccount userAccount, Erro error]
    UserAccount userAccount;
}

class LoginRequest{
    String user;
    String password;
}

class LoginResponse{
    ApiPostLoginResponse apiPostLoginResponse;

}

File: app/src/main/java/com/br/natanfelipe/bankapp/view/login/LoginModel.kt
package com.br.natanfelipe.bankapp.view.login

import com.br.natanfelipe.bankapp.api.RestApi
import com.br.natanfelipe.bankapp.model.UserAccount

class LoginModel {
}

class LoginViewModel {
    var userAccount : UserAccount = UserAccount(0,"","",0,0.0)
}

class LoginRequest() {
    var api = RestApi()
}

class LoginResponse() {
    var userAccount : UserAccount = UserAccount(0,"","",0,0.0)
}